pkg/invoke/invoker.go

```diff
@@ -35,24 +35,26 @@ import (
 var log = logger.Package()
 
 type Invoker struct {
-	gptClient     *gptscript.GPTScript
-	uncached      kclient.WithWatch
-	gatewayClient *client.Client
-	tokenService  *jwt.TokenService
-	events        *events.Emitter
-	serverURL     string
-	serverPort    int
+	gptClient           *gptscript.GPTScript
+	uncached            kclient.WithWatch
+	gatewayClient       *client.Client
+	tokenService        *jwt.TokenService
+	events              *events.Emitter
+	noReplyEmailAddress string
```

Comment:
should this configuration name specifically mention SendGrid; e.g. 'sendGridNoReplyAddress`?

Should it even be a full address? Maybe something like `emailDomain` (to match the version API) is better, and we just use a default address like `obot-noreply@<emailDomain>`?

Reply:
I don't think it should specifically mention SendGrid. It is likely this would be useful for other similar tools we might add in the future.

I do like the idea of just using an `emailDomain` rather than requiring them to specify the full address, but I personally think it is fine either way.